src/model/database.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+from model.table import Table
+from commands.keys import Keys
+from output.exceptions import *
+
+
+class Database:
+
+    def __init__(self, schema_name):
+        self.__get_database_schema__(schema_name)
+        self.name = self.database_object[Keys.DATABASE]
+        self.__validate_name__()
```

Comment:
it is better to call this method validate, and validate everything. if u want to validate later something else, u can put it in validate